apps/ejabberd/src/ejabberd_local_delivery.erl

```diff
@@ -0,0 +1,34 @@
+%%%-------------------------------------------------------------------
```

Comment:
Yes I know, I already asked MP what should be put here - what license etc. Files in source tree have various headers. Thaks for the reminder.